WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/util/analytics/WooPosAnalyticsEventConstant.kt

```diff
@@ -1,9 +1,83 @@
 package com.woocommerce.android.ui.woopos.util.analytics
 
 object WooPosAnalyticsEventConstant {
-    const val PRODUCT_TYPE = "product_type"
-    const val ITEM_LIST_TYPE = "item_list_type"
-    const val ITEM_LIST_TYPE_PRODUCTS = "products"
-    const val ITEM_LIST_TYPE_VARIATIONS = "variations"
-    const val IS_SEARCH = "search"
+    enum class ItemsListProductType(val value: String) {
+        SIMPLE("simple"),
+        VARIABLE("variable");
```

Comment:
We select a variable product to open a list of variations. But then we add a variation to the cart (my understanding).

```suggestion
        VARATION("variation");
```

Reply:
You are correct, good catch! Fixed